router/server.go

```diff
@@ -61,6 +61,8 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 		Methods("GET")
 	gMux.Handle("/api/perf/result", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetResultHandler)))).
 		Methods("GET")
+	gMux.Handle("/api/perf/meshes", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetSMPServiceMeshes)))).
```

Comment:
Can we rename it to something more descriptive `api/perf/mesh/list`?

Reply:
Oh yeah, that makes more sense. Thanks!